test/new-e2e/system-probe/system-probe-test-env.go

```diff
@@ -206,7 +206,7 @@ func NewTestEnv(name, x86InstanceType, armInstanceType string, opts *EnvOpts) (*
 		runner.AWSKeyPairName:            auto.ConfigValue{Value: opts.SSHKeyName},
 		// Its fine to hardcode the password here, since the remote ec2 instances do not have
 		// any password on sudo. This secret configuration was introduced in the test-infra-definitions
-		// scenario for dev environments: https://github.com/DataDog/test-infra-definitions/pull/159
+		// scenario for dev environments: https://github.com/DataDog/datadog-agent/test/e2e-framework/pull/159
```

Comment:
nit: no need to change this URL :D 
```suggestion
		// scenario for dev environments: https://github.com/DataDog/test-infra-definitions/pull/159
```